dali/ft/daftformat.cpp

```diff
@@ -702,19 +702,33 @@ void CCsvPartitioner::getRecordStructure(StringBuffer & _recordStructure)
     if (nullptr == strstr(recordStructure.str(),"END;"))
     {
         const byte *buffer = bufferBase();
-        ensureBuffered(headerSize);
-        assertex((headerSize == 0) || (numInBuffer != bufferOffset));
+        unsigned processSize = bufferSize;
+
+        if (!ensureBuffered(headerSize) && (numInBuffer < headerSize))
+        {
+            // This means the stream/file is shortest than the headerSize, however it can hold
+            // valid records
+            LOG(MCdebugProgressDetail, unknownJob, "It is possible the file is shorter (%d bytes) than the header size(%d bytes)", numInBuffer, headerSize);
```

Comment:
Makes sense I think, file shorter than maxrecord size basically..
I don't think logging this is appropriate though, a comment would suffice here.